### PR TITLE
[Website]: Update "Getting started" developers content

### PR DESCRIPTION
**Pull request by maya**
## Description

Updates "Getting started" developers content.

### CODE REVIEW

**Comment by msecret** [comment]
LGTM

File: docs/pages/getting-started/developers.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ You can also email us directly at uswebdesignstandards@gsa.gov.
 * Uses a **[modified BEM](https://pages.18f.gov/frontend/css-coding-styleguide/naming/)** approach created by 18F for naming CSS selectors. Objects in CSS are separated by single dashes. Multi-word objects are separated by an underscore (For example: `.usa-button-cool_feature-active`).
 * Uses **modular CSS** for scalable, modular, and flexible code.
 * Uses **nesting** when appropriate. Nest minimally with up to two levels of nesting.
-* **Global variables** are defined in the `_defaults.scss` file. Custom theming can be done by copying `_variables.scss`, changing variable values and importing it before uswds in Sass.
+* **Global variables** are defined in the `_defaults.scss` file. Custom theming can be done by copying `_variables.scss` into your own project's Sass folder, changing variable values, and importing it before uswds in Sass.
```
Comment (kategarklavs) [inline docs/pages/getting-started/developers.md]: @maya Would it make sense to say "changing the relevant variable values," or is that overkill?

**Comment by kategarklavs** [comment]
@maya Just left you a line note re: a particular phrasing. Otherwise, this looks good to me!

File: docs/pages/getting-started/developers.md (hunk 1 above)
Comment (maya) [inline docs/pages/getting-started/developers.md]: @kategarklavs perhaps! what we're trying to say is they can change whichever variables apply to them for their project's needs. Does "relevant" infer that?

Also what do you think about modifying "copying `_variables.scss`" slightly to: 
>copying the `_variables.scss` file...

Comment (kategarklavs) [inline docs/pages/getting-started/developers.md]: Hey! _Relevant_ works, but _applicable_ might be even more precise 👍

**Comment by kategarklavs** [comment]
🐭 !

**Pull request merged**